src/runtime/test/manifest-test.ts

```diff
@@ -1990,6 +1990,25 @@ resource SomeName
       assert.equal(claim2.tag, 'property2');
     });
 
+    it('supports "is not" tag claims', async () => {
+      const manifest = await Manifest.parse(`
+        particle A
+          out T {} output1
+          out T {} output2
+          claim output1 is not property1
+      `);
+      assert.lengthOf(manifest.particles, 1);
+      const particle = manifest.particles[0];
+      assert.isEmpty(particle.trustChecks);
+      assert.equal(particle.trustClaims.size, 1);
+
+      const claim1 = particle.trustClaims.get('output1') as ClaimIsTag;
+      assert.equal(claim1.handle.name, 'output1');
+      assert.equal(claim1.isNot, true);
+      assert.equal(claim1.tag, 'property1');
+
+     });
+
     it('supports "derives from" claims with multiple parents', async () => {
       const manifest = await Manifest.parse(`
         particle A
```

Comment:
Can you add a line to the "can round-trip checks and claims" test in this file below, to check that "not" gets preserved?